static/js/add-to-calendar.js

```diff
@@ -0,0 +1,486 @@
+;(function(exports) {
```

Comment:
from https://github.com/commonpike/add-to-calendar-buttons/blob/master/add-to-calendar.js
with some modification on Line 16--20 to use shorter names